docker-compose.yml

```
@@ -0,0 +1,44 @@
+    build:
+      context: .
+      dockerfile: docker/Dockerfile.mlflow
+    container_name: mlflow
+    ports:
+      - "5001:5000"
+    networks:
+      - backend
+
+networks:
+  backend:
+    driver: bridge 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)